libgengo/context.go

```
@@ -238,9 +239,11 @@ func (ctx *PkgContext) LoadSrvFromString(text string, fullname string) (*SrvSpec
 		return nil, err
 	}
 
-	components := strings.Split(text, "---")
+	rex := regexp.MustCompile("[-]{3,}")
```

MustCompile panics instead of returning an error: https://gobyexample.com/regular-expressions

Is this an intended behaviour?

---

Yes; because it's compiling a literal, it either panics during testing, or never panics.